taxi/views.py

```diff
@@ -1,3 +1,15 @@
 from django.shortcuts import render
+from taxi.models import Driver, Manufacturer, Car
 
-# Create your views here.
+
+def index(request):
+    num_drivers = Driver.objects.count()
+    num_manufacturers = Manufacturer.objects.count()
+    num_cars = Car.objects.count()
```

Comment:
there is no need to create additional variables here

Reply:
Could you please advise, on the correct way to configure the Secret Key for GitHub